src/ExtensionsHostLib/RawProfilerHookSettingsReader.cpp

```diff
@@ -12,8 +12,13 @@ namespace Agent
 {
 namespace Host
 {
-    LPCWSTR cszRawProfilerHookVariableName = L"MicrosoftInstrumentationEngine_RawProfilerHook";
-    LPCWSTR cszRawProfilerHookPathVariableName = L"MicrosoftInstrumentationEngine_RawProfilerHookPath";
```

Comment:
This is technically a breaking change, but I don't know how useful this was previously considering that it could try to load the assembly with a different bitness and fail. Would it be useful to check for the variable with the bitness in its name first and then fallback to the one without the bitness in its name, to preserve backward compatibility?